js/form.js

```diff
@@ -30,3 +46,31 @@ formAd.addEventListener('submit', (evt) => {
     resetForm(evt);
   }
 });
+
+
+avatarChooser.addEventListener('change', () => {
+  const avatar = avatarChooser.files[0];
+  const fileName = avatar.name.toLowerCase();
+  const matches = FILE_TYPES.some((it) => fileName.endsWith(it));
+  if (matches) {
+    avatarPreview.src = URL.createObjectURL(avatar);
+  }
+});
+
+photoChooser.addEventListener('change', () => {
+  const photo = photoChooser.files[0];
+  const photoName = photo.name.toLowerCase();
+  const exist = document.querySelector('.ad-form__photo img');
+  const photoElement = document.createElement('img');
+  photoElement.style.width = '70px';
+  photoElement.style.height = '70px';
+  const matches = FILE_TYPES.some((it) => photoName.endsWith(it));
+  if (matches) {
+    if (exist) {
```

Comment:
```suggestion
  if (matches) {
    return
  }
  
 if (exist) {

```
старайся избегать вложенных ифов